caravel/assets/javascripts/reduxUtils.js

```diff
@@ -94,3 +94,18 @@ export function areArraysShallowEqual(arr1, arr2) {
   }
   return true;
 }
+
+export function areObjectsEqual(obj1, obj2) {
```

Comment:
I think there's a missing case here where obj2 has properties that are not in obj1.